### DISCUSSION TITLE
p

### DISCUSSION THREAD

**Discussion by ghost** [smartcontractkit/full-blockchain-solidity-course-py#281] [Q&A]

**Comment by cromewar** [comment]
Hello @balabala7. could you please be more specific with your question?

Anyways this are some resources that might help you:

[How to Create a Generative NFT project](https://www.youtube.com/watch?v=ftMzXpToq9Q)

[How to bring your NFT project to market in days with NFTPort, the Stripe for NFTs](https://www.youtube.com/watch?v=MCvoSbdgooE)

[Moralis | Fastest way to build dapps fast](https://www.youtube.com/watch?v=NDRMr2uPcIA)

Best regards, Cromewar.
